src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -346,6 +341,7 @@ class GenerateHierarchyFunctor
   KOKKOS_INLINE_FUNCTION
   void operator()(int const i) const
   {
+    auto const root = _internal_nodes.data();
```

Comment:
Where is `root` used??

Reply:
I guess it's not, and is just an artifact of your history when working with `node_ptr` patch.